bench/criterion/criterion_benchmark.rs

```diff
@@ -1,11 +1,21 @@
 use cleopatra_cairo::cairo_run;
 use criterion::{black_box, criterion_group, criterion_main, Criterion};
 
-pub fn criterion_benchmark(c: &mut Criterion) {
+pub fn criterion_benchmark_fibonacci(c: &mut Criterion) {
     c.bench_function("cairo_run(bench/criterion/fibonacci_1000.json", |b| {
         b.iter(|| cairo_run::cairo_run(black_box("bench/criterion/fibonacci_1000.json")))
     });
 }
 
-criterion_group!(benches, criterion_benchmark);
+pub fn criterion_benchmark_integration(c: &mut Criterion) {
+    c.bench_function("cairo_run(bench/criterion/integration.json", |b| {
+        b.iter(|| cairo_run::cairo_run(black_box("bench/criterion/fibonacci_1000.json")))
```

Comment:
@fmoletta why do we have integration as fibonacci + integration and fibonacci as a separate test?